app/src/main/java/org/apache/fineract/ui/online/accounting/accounts/AccountsFragment.kt

```diff
@@ -164,4 +167,13 @@ class AccountsFragment : FineractBaseFragment(), AccountContract.View, SwipeRefr
         accountsPresenter.detachView()
     }
 
+    override fun onBackPressed(): Boolean {
+        if (searchView?.isIconified!!)
```

Comment:
`!!` causes the app to crash if the object is null, instead of this it would be much better to s.do something like this: `searchView?.isIconified == true`